augly/tests/image_tests/functional_unit_test.py

```diff
@@ -107,7 +108,7 @@ def test_random_noise(self):
         self.evaluate_function(imaugs.random_noise)
 
     def test_resize(self):
-        self.evaluate_function(imaugs.resize)
+        self.evaluate_function(imaugs.resize, resample=Image.BICUBIC)
```

Comment:
This is good, but please make this change to the transforms unit test too! That one is currently failing in the `test_python` workflow since it's being run with the default resample method `Image.BILINEAR`, whereas the test output file was generated by running with `Image.BICUBIC`.

Reply:
Note: you'll also need to add the `resample` param to the expected metadata for the HFlip transforms unit test. You can first add the `resample=Image.BICUBIC` to the test itself like you did here, then run the test, then the output should show you the new expected metadata with the `resample` param.